NEWS.md

```diff
@@ -5,6 +5,16 @@ NEW FUNCTIONS
 * `logSafe()` to compute logarithm of values that could be close to 0 or slightly 
 negative.
 
+* `foldSafe()` to compute `x / y` when `x` or `y` could be negative or zero. All values below a 
+certain threshold `epsilon` are substituted by `epsilon`. NOTE: not suited for 
+calculating fold differences of negative numbers.
```

Comment:
Theoretically, we could add an optional boolean argument telling if the compared values are supposed to be negative.
If so, we would replace all values which are `>-epsilon` with `-epsilon`.
But I am not aware of any practical use case for this :)